app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+from typing import List
```

Comment:
u didn't fix that
https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be,group%20of%20imports.